agent/utils.py

```diff
@@ -10,6 +10,8 @@
 from ostorlab.agent.kb import kb
 from ostorlab.agent.mixins import agent_report_vulnerability_mixin
 
+LINE_SIZE_MAX = 5000
```

Comment:
we can miss the location of the vulnerability this way , for example if it's in the 5001 line we will miss it

Reply:
can't we locate it and then take a range like before and after this line

Reply:
Correct, hence I use beautifier to reduce the chances of this happening. The rest is will in anyway be bad because the code will truncated in the middle.